src/generate-id.js

```diff
@@ -1,4 +1,13 @@
 /* @flow */
 
-export const generateId = () : string =>
-    [ Math.random(), Math.random() ].map(x => x.toString(16).slice(2)).join('');
+/*
+** the reason this is exported as a default object is to get rid of error
+** "Cannot set property "prop" of #<Object> which has only a getter"
+** - flow and eslint have to be disabled on whatever line this is used.
+*/
+
+// eslint-disable-next-line import/no-default-export
+export default {
+    generateId: () : string =>
```

Comment:
If you like, you can `import { uniqueID } from 'belter/src';` -- this is the function we use throughout the rest of the sdk.